python/sqlflow_submitter/db.py

```diff
@@ -155,22 +156,21 @@ def read_feature(raw_val, feature_spec, feature_name):
         values = np.ones([indices.size], dtype=np.int32)
         dense_shape = np.array(feature_spec["shape"], dtype=np.int64)
         return (indices, values, dense_shape)
-    else:
+    elif feature_spec["delimiter"] != "":
```

Comment:
The condition `feature_spec["delimiter"] != ""` is not mutually exclusive with the one of `if feature_spec["is_sparse"]` which makes this condition hard to read. I think keep the previous  code may be easy to understand.